app/views/user_mailer/shared/_in_person_warning_banner.html.erb

```diff
@@ -0,0 +1,11 @@
+<table class="usa-alert usa-alert--warning margin-bottom-4">
+  <tbody>
+  <tr>
+    <td style="width:16px;">
+      <%= image_tag('email/warning.png', width: 16, height: 14, alt: '', style: 'margin-top: 5px;') %>
```

Comment:
I know there are many instances where alt text is blank but I still prefer we don't leave it blank. When I test with VoiceOver, it recognizes the image but our users don't have any context or knowledge about it. I think if we have an image, we should have alt text for our uses and to be 508 compliant, see GSA resource [here](https://www.section508.gov/training/alt-text/what-is-alternative-text/).

Reply:
done